views/Game.jsx

```diff
@@ -3,55 +3,50 @@ import PlayerArea from "../components/PlayerArea";
 import Dot from "../components/Dot";
 import { Alert } from "react-native";
 
-const difficulties = {
-  EASY: "Easy",
-  MEDIUM: "Medium",
-  HARD: "Hard"
-};
-
-const difficultyModifier = {
-  [difficulties.EASY]: 3,
-  [difficulties.MEDIUM]: 2,
-  [difficulties.HARD]: 1
-};
-
-const difficulty = difficulties.EASY;
-
-const playerModifier = {
-  0: currentDot => currentDot - difficultyModifier[difficulty],
-  1: currentDot => currentDot + difficultyModifier[difficulty]
-};
-
-function Game() {
-  const [score, setScore] = useState(0);
+function Game(props) {
+  const [position, setPosition] = useState(0);
 
   const showWinnerMessage = winner => {
     Alert.alert(`Player ${winner} Wins`, "", [
       {
         text: "Start Over",
         onPress: () => {
-          setScore(0);
+          setPosition(0);
+        }
+      },
+      {
+        text: "Back To Nav",
+        onPress: () => {
+          setPosition(0)
+          props.setHidden();
         }
       }
     ]);
   };
-  if (score <= -36) {
+
+
+  if (position <= -36) {
     showWinnerMessage(1);
-  } else if (score >= 36) {
+  } else if (position >= 36) {
     showWinnerMessage(2);
   }
 
   const onPlayerTap = player => {
-    const newScore = playerModifier[player](score);
-    setScore(newScore);
+    if (player === 'playerOne') {
+      setPosition(position - props.playerDifficulty.playerOne);
+    } else if (player === 'playerTwo') {
+      setPosition(position + props.playerDifficulty.playerTwo);
+    }
   };
+
   return (
     <React.Fragment>
-      <PlayerArea player={0} onTap={onPlayerTap} />
-      <PlayerArea player={1} onTap={onPlayerTap} />
-      <Dot offset={score * 10} />
+      <PlayerArea player={'playerOne'} onTap={onPlayerTap} />
```

Comment:
To the same comment above, let's not use raw strings. Either store it in a variable or import it from another file